### ISSUE TITLE
UI issues for tablet

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#5882]
**Bug description:**
UI issues for tablet

**Actual result:**
Back option not showing properly
![back](https://github.com/opencrvs/opencrvs-core/assets/94350336/d150d467-acf1-4802-a7af-d1099e358ea4)

in correction fee document- delete option not placed properly
![ui in tab](https://github.com/opencrvs/opencrvs-core/assets/94350336/18a786a7-7f62-4363-b0b5-8e77df4929ea)

 cannot see full modal of signature
![CANNOT SEE FUL MODAL OF SIGT](https://github.com/opencrvs/opencrvs-core/assets/94350336/e40cfd78-2bd3-4c95-9c94-fb918922a9a1)

**Tested on:**
browserstack- Galaxy Tab S7

**Comment by SyedaAfrida** [comment]
The mentioned bugs are not fixed yet.

**Comment by Nil20** [comment]
@jpye-finch Do you have any input on how it should be or going to be? And I think we should start working on fixing it after we have a conclusive decision about this.

**Comment by jpye-finch** [comment]
1. For the first issue/image. Applying the solution outlined in the ticket for tablet potrait. Will fix the issue with the back button
https://app.zenhub.com/workspaces/opencrvs-core-5afeded64a54b06b54a969b2/issues/gh/opencrvs/opencrvs-core/5915

2. Need to look at DocumentListViewer to make it responsive

4. Using the new Dialog component instead of ResponsiveModel should fix this.

**Comment by euanmillar** [comment]
Blocking as the 3 tickets JPF mentioned address this and should be prioritised first

**Comment by eduffus** [comment]
To be retested after #5915 complete

**Comment by euanmillar** [comment]
@eduffus you marked this to be re-tested after #5915 is complete which is a UI issue from 2023 and has no milestone.  This is also marked as low priority.  Therefore this should be moved out of the 1.5.0 release and addressed by new developers.
